### PR TITLE
Change default write type to async through

### PR DESCRIPTION
**Pull request by calvinjia**
Async through is a more reasonable default write type.

### CODE REVIEW

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: docs/en/basic/Configuration-Settings.md [hunk 1]
```diff
@@ -131,12 +131,11 @@ or worker settings (e.g., `alluxio.worker.*`)
...
 which will be distributed and become cluster-wide default values when clients and workers connect.
 
-For example, the property `alluxio.user.file.writetype.default` defaults to `MUST_CACHE`,
-which only writes to Alluxio space.
-In an Alluxio cluster where data persistence is preferred
-and all jobs need to write to both the UFS and Alluxio,
-the administrator can add `alluxio.user.file.writetype.default=CACHE_THROUGH` in each master's
-`alluxio-site.properties` file.
+For example, the property `alluxio.user.file.writetype.default` defaults to `ASYNC_THROUGH`, which
+first writes to Alluxio and then asynchronously writes to the UFS.
```
Comment (gpang) [inline docs/en/basic/Configuration-Settings.md]: do we need to mention `alluxio.user.file.replication.durable` here?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -2840,11 +2840,12 @@ public String toString() {
...
   public static final PropertyKey USER_FILE_WRITE_TYPE_DEFAULT =
       new Builder(Name.USER_FILE_WRITE_TYPE_DEFAULT)
-          .setDefaultValue("MUST_CACHE")
+          .setDefaultValue("ASYNC_THROUGH")
           .setDescription("Default write type when creating Alluxio files. Valid options are "
               + "`MUST_CACHE` (write will only go to Alluxio and must be stored in Alluxio), "
               + "`CACHE_THROUGH` (try to cache, write to UnderFS synchronously), `THROUGH` "
-              + "(no cache, write to UnderFS synchronously).")
+              + "(no cache, write to UnderFS synchronously), `ASYNC_THROUGH` (write to cache, "
+              + "write to UnderFS synchronously.")
```
Comment (gpang) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java]: Should we mention the relationship with `alluxio.user.file.replication.durable` here?

Comment (calvinjia) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: Updated, thanks.

File: docs/en/basic/Configuration-Settings.md (hunk 1 above)
Comment (calvinjia) [reply docs/en/basic/Configuration-Settings.md]: Autogenerated

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**